.github/workflows/publish-snapshot.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Snapshot
+
+on:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  publish-snapshot:
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: ubuntu-latest
```

Comment:
shall we use the latest LTS？

Reply:
Make sense, I'm going to switch all image to `ubuntu-20.04`. BTW, Apache Spark mix uses `ubuntu-20.04` and `ubuntu-latest`.